### PR TITLE
Marge treasure data v0.6.24

### PR DESCRIPTION
**Pull request by t3t5u**
### Are all connections created by the plugin secure?

- [ ] Does it opt secure communication standard? Such as HTTPS, SSH, SFTP, SMTP STARTTLS. If not check with CISO to decide we can deploy the plugin.
- [ ] Does support both authentication and encryption appropriately? Such as: "just encrypting without authentication" that is insecure.

### Does the plugin connect only to its expected external site which the customer explicitly set in their config file?

- [ ] Does NOT connect unexpected external site and our internal endpoints? Such as: “v3/job/:id/set_started” callback endpoint.

### Does NOT the plugin persist any customers' private information? Identify the private information beforehand.

- [ ] Does NOT include them in (temporary) files?
- [ ] Does NOT include them in log messages and exception messages?

### What kind of environments does the plugin interact with?

- [ ] Does NOT execute any shell command?
- [ ] Does NOT read any files on the running instance? Such as: "/etc/passwords". It’s ok to read temporary files that the plugin wrote.
- [ ] Does use to create temporary files by spi.TempFileSpace utility to avoid the conflict of the file names.
- [ ] Does NOT get environment variables or JVM system properties at runtime? Such as AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY in environment variables

### Does NOT the plugin use insecure libraries?

- [ ] Line up all depending library so that we can identify the impact of security incident of those library if any.
- [ ] Check libraries usage of the plugin; all security check list must apply to the library usages. Such as "Are all connections created by the library secure?"

### Does NOT the plugin source code repository contain kinds of credentials

- [ ] API keys
- [ ] Passwords

### Make sure to free up all resources allocated during Embulk transaction “committing” or “rolling back”or before.

- [ ] Network (connections, pooled connections)
- [ ] Memory (cache in static variables)
- [ ] File (temporary files)
- [ ] CPU (threads, processes)

### CODE REVIEW

File: src/main/java/org/embulk/input/marketo/delegate/MarketoBaseBulkExtractInputPlugin.java [hunk 1]
```diff
@@ -196,63 +187,6 @@ private CSVParser getCsvParser(LineDecoderIterator decoderIterator) throws IOExc
         return csvTokenizer.csvParse();
     }
 
-    /**
-     * This method should be removed when we allow skip preview phase
-     */
-    private TaskReport importMockPreviewData(final PageBuilder pageBuilder)
```
Comment (t3t5u) [inline src/main/java/org/embulk/input/marketo/delegate/MarketoBaseBulkExtractInputPlugin.java:202]: src/main/java/org/embulk/input/marketo/MarketoUtils.java に移動しています

File: src/main/java/org/embulk/input/marketo/delegate/MarketoBaseBulkExtractInputPlugin.java [hunk 2]
```diff
@@ -272,99 +206,6 @@ protected final Iterator<ServiceRecord> getServiceRecords(MarketoService marketo
 
     protected abstract InputStream getExtractedStream(MarketoService service, T task, OffsetDateTime fromDate, OffsetDateTime toDate);
 
-    private static class AllStringJacksonServiceRecord extends JacksonServiceRecord
```
Comment (t3t5u) [inline src/main/java/org/embulk/input/marketo/delegate/MarketoBaseBulkExtractInputPlugin.java:275]: src/main/java/org/embulk/input/marketo/bulk_extract/AllStringJacksonServiceRecord.java に移動しています

File: build.gradle [hunk 3]
```diff
@@ -15,14 +15,14 @@ repositories {
...
     def vd = versionDetails()
-    if (vd.lastTag != "${baseVersion}.${patchVersion}") {
-        logger.warn "lastTag '${vd.lastTag}' is not '${baseVersion}.${patchVersion}'"
+    if (vd.lastTag != "v${baseVersion}.${patchVersion}") {
+        logger.warn "lastTag '${vd.lastTag}' is not 'v${baseVersion}.${patchVersion}'"
     }
-    if (vd.commitDistance == 0 && vd.lastTag ==~ /^[0-9]+\.[0-9]+\.[0-9]+(\.[a-zA-Z0-9]+)?/) {
-        "${vd.lastTag}.trocco"
+    if (vd.commitDistance == 0 && vd.lastTag ==~ /^v[0-9]+\.[0-9]+\.[0-9]+(\.[a-zA-Z0-9]+)?/) {
+        "${vd.lastTag.replaceFirst(/^v/, "")}.trocco"
```
Comment (t3t5u) [inline build.gradle]: 以下の対応
https://prime-number.slack.com/archives/C01R5FL2MFY/p1656488542767229?thread_ts=1656306084.556129&cid=C01R5FL2MFY

version 自体には `v` を付けず,
タグ名には `v` を付けるようにしました

**Comment by t3t5u** [comment]
# 動作確認

## embulk preview:

```
$ embulk preview config.yml
2022-07-01 20:16:54.947 +0900: Embulk v0.9.26
2022-07-01 20:16:55.540 +0900 [WARN] (main): DEPRECATION: JRuby org.jruby.embed.ScriptingContainer is directly injected.
2022-07-01 20:16:56.459 +0900 [INFO] (main): Gem's home and path are set by default: "/home/ec2-user/.embulk/lib/gems"
2022-07-01 20:16:56.874 +0900 [INFO] (main): Started Embulk v0.9.26
2022-07-01 20:16:56.939 +0900 [INFO] (0001:preview): Loaded plugin embulk-input-marketo (0.6.24.1361d01801.pre)
2022-07-01 20:16:56.986 +0900 [INFO] (0001:preview): Logging initialized @2239ms
2022-07-01 20:16:57.833 +0900 [INFO] (0001:preview): Acquired new access token
2022-07-01 20:16:57.833 +0900 [INFO] (0001:preview): CALLING Account ID: 498-VPN-295 GET -> https://498-VPN-295.mktorest.com/rest/asset/v1/programs.json - params: {offset=[0], maxReturn=[200], filterType=[id], filterValues=[1635], batchSize=[300]}
2022-07-01 20:16:58.081 +0900 [INFO] (0001:preview): CALLING Account ID: 498-VPN-295 GET -> https://498-VPN-295.mktorest.com/rest/v1/programs/members/describe.json - params: null
+------------------------+----------------------+---------------------+------------------------------+-------------------+---------------------------------+---------------------+-----------------------+-----------------------+-----------------------+-----------------------------+-----------------------------+------------------------+--------------------------------+--------------------------+----------------------+----------------------------+------------------------------+--------------------------+-------------------+---------------------------+----------------+-----------------------------+
| mk_isExhausted:boolean | mk_webinarUrl:string | mk_prog_custf2:long | mk_programmembertest1:string | mk_program:string | mk_reachedSuccessDate:timestamp | mk_trackName:string | mk_prog_custf3:string | mk_prog_custf1:string | mk_acquiredBy:boolean | mk_membershipDate:timestamp |      mk_createdAt:timestamp | mk_statusReason:string | mk_registrationLikelihood:long | mk_nurtureCadence:string | mk_statusName:string | mk_registrationCode:string | mk_attendanceLikelihood:long | mk_waitlistPriority:long | mk_programId:long | mk_reachedSuccess:boolean | mk_leadId:long |      mk_updatedAt:timestamp |
+------------------------+----------------------+---------------------+------------------------------+-------------------+---------------------------------+---------------------+-----------------------+-----------------------+-----------------------+-----------------------------+-----------------------------+------------------------+--------------------------------+--------------------------+----------------------+----------------------------+------------------------------+--------------------------+-------------------+---------------------------+----------------+-----------------------------+
|                  false |      mk_webinarUrl_1 |              12,345 |      mk_programmembertest1_1 |      mk_program_1 |     2022-07-01 11:16:58.398 UTC |      mk_trackName_1 |      mk_prog_custf3_1 |      mk_prog_custf1_1 |                 false | 2022-07-01 11:16:58.398 UTC | 2022-07-01 11:16:58.398 UTC |      mk_statusReason_1 |                         12,345 |      mk_nurtureCadence_1 |      mk_statusName_1 |      mk_registrationCode_1 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.398 UTC |
|                  false |      mk_webinarUrl_2 |              12,345 |      mk_programmembertest1_2 |      mk_program_2 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_2 |      mk_prog_custf3_2 |      mk_prog_custf1_2 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_2 |                         12,345 |      mk_nurtureCadence_2 |      mk_statusName_2 |      mk_registrationCode_2 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |      mk_webinarUrl_3 |              12,345 |      mk_programmembertest1_3 |      mk_program_3 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_3 |      mk_prog_custf3_3 |      mk_prog_custf1_3 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_3 |                         12,345 |      mk_nurtureCadence_3 |      mk_statusName_3 |      mk_registrationCode_3 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |      mk_webinarUrl_4 |              12,345 |      mk_programmembertest1_4 |      mk_program_4 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_4 |      mk_prog_custf3_4 |      mk_prog_custf1_4 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_4 |                         12,345 |      mk_nurtureCadence_4 |      mk_statusName_4 |      mk_registrationCode_4 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |      mk_webinarUrl_5 |              12,345 |      mk_programmembertest1_5 |      mk_program_5 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_5 |      mk_prog_custf3_5 |      mk_prog_custf1_5 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_5 |                         12,345 |      mk_nurtureCadence_5 |      mk_statusName_5 |      mk_registrationCode_5 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |      mk_webinarUrl_6 |              12,345 |      mk_programmembertest1_6 |      mk_program_6 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_6 |      mk_prog_custf3_6 |      mk_prog_custf1_6 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_6 |                         12,345 |      mk_nurtureCadence_6 |      mk_statusName_6 |      mk_registrationCode_6 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |      mk_webinarUrl_7 |              12,345 |      mk_programmembertest1_7 |      mk_program_7 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_7 |      mk_prog_custf3_7 |      mk_prog_custf1_7 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_7 |                         12,345 |      mk_nurtureCadence_7 |      mk_statusName_7 |      mk_registrationCode_7 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |      mk_webinarUrl_8 |              12,345 |      mk_programmembertest1_8 |      mk_program_8 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_8 |      mk_prog_custf3_8 |      mk_prog_custf1_8 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_8 |                         12,345 |      mk_nurtureCadence_8 |      mk_statusName_8 |      mk_registrationCode_8 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |      mk_webinarUrl_9 |              12,345 |      mk_programmembertest1_9 |      mk_program_9 |     2022-07-01 11:16:58.399 UTC |      mk_trackName_9 |      mk_prog_custf3_9 |      mk_prog_custf1_9 |                 false | 2022-07-01 11:16:58.399 UTC | 2022-07-01 11:16:58.399 UTC |      mk_statusReason_9 |                         12,345 |      mk_nurtureCadence_9 |      mk_statusName_9 |      mk_registrationCode_9 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.399 UTC |
|                  false |     mk_webinarUrl_10 |              12,345 |     mk_programmembertest1_10 |     mk_program_10 |     2022-07-01 11:16:58.400 UTC |     mk_trackName_10 |     mk_prog_custf3_10 |     mk_prog_custf1_10 |                 false | 2022-07-01 11:16:58.400 UTC | 2022-07-01 11:16:58.400 UTC |     mk_statusReason_10 |                         12,345 |     mk_nurtureCadence_10 |     mk_statusName_10 |     mk_registrationCode_10 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.400 UTC |
|                  false |     mk_webinarUrl_11 |              12,345 |     mk_programmembertest1_11 |     mk_program_11 |     2022-07-01 11:16:58.400 UTC |     mk_trackName_11 |     mk_prog_custf3_11 |     mk_prog_custf1_11 |                 false | 2022-07-01 11:16:58.400 UTC | 2022-07-01 11:16:58.400 UTC |     mk_statusReason_11 |                         12,345 |     mk_nurtureCadence_11 |     mk_statusName_11 |     mk_registrationCode_11 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.400 UTC |
|                  false |     mk_webinarUrl_12 |              12,345 |     mk_programmembertest1_12 |     mk_program_12 |     2022-07-01 11:16:58.400 UTC |     mk_trackName_12 |     mk_prog_custf3_12 |     mk_prog_custf1_12 |                 false | 2022-07-01 11:16:58.400 UTC | 2022-07-01 11:16:58.400 UTC |     mk_statusReason_12 |                         12,345 |     mk_nurtureCadence_12 |     mk_statusName_12 |     mk_registrationCode_12 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.400 UTC |
|                  false |     mk_webinarUrl_13 |              12,345 |     mk_programmembertest1_13 |     mk_program_13 |     2022-07-01 11:16:58.400 UTC |     mk_trackName_13 |     mk_prog_custf3_13 |     mk_prog_custf1_13 |                 false | 2022-07-01 11:16:58.400 UTC | 2022-07-01 11:16:58.400 UTC |     mk_statusReason_13 |                         12,345 |     mk_nurtureCadence_13 |     mk_statusName_13 |     mk_registrationCode_13 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.400 UTC |
|                  false |     mk_webinarUrl_14 |              12,345 |     mk_programmembertest1_14 |     mk_program_14 |     2022-07-01 11:16:58.400 UTC |     mk_trackName_14 |     mk_prog_custf3_14 |     mk_prog_custf1_14 |                 false | 2022-07-01 11:16:58.400 UTC | 2022-07-01 11:16:58.400 UTC |     mk_statusReason_14 |                         12,345 |     mk_nurtureCadence_14 |     mk_statusName_14 |     mk_registrationCode_14 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.400 UTC |
|                  false |     mk_webinarUrl_15 |              12,345 |     mk_programmembertest1_15 |     mk_program_15 |     2022-07-01 11:16:58.400 UTC |     mk_trackName_15 |     mk_prog_custf3_15 |     mk_prog_custf1_15 |                 false | 2022-07-01 11:16:58.400 UTC | 2022-07-01 11:16:58.400 UTC |     mk_statusReason_15 |                         12,345 |     mk_nurtureCadence_15 |     mk_statusName_15 |     mk_registrationCode_15 |                       12,345 |                   12,345 |            12,345 |                     false |         12,345 | 2022-07-01 11:16:58.400 UTC |
+------------------------+----------------------+---------------------+------------------------------+-------------------+---------------------------------+---------------------+-----------------------+-----------------------+-----------------------+-----------------------------+-----------------------------+------------------------+--------------------------------+--------------------------+----------------------+----------------------------+------------------------------+--------------------------+-------------------+---------------------------+----------------+-----------------------------+
```

## embulk run:

```
$ embulk run config.yml
2022-07-01 20:17:18.552 +0900: Embulk v0.9.26
2022-07-01 20:17:19.164 +0900 [WARN] (main): DEPRECATION: JRuby org.jruby.embed.ScriptingContainer is directly injected.
2022-07-01 20:17:20.411 +0900 [INFO] (main): Gem's home and path are set by default: "/home/ec2-user/.embulk/lib/gems"
2022-07-01 20:17:21.029 +0900 [INFO] (main): Started Embulk v0.9.26
2022-07-01 20:17:21.111 +0900 [INFO] (0001:transaction): Loaded plugin embulk-input-marketo (0.6.24.1361d01801.pre)
2022-07-01 20:17:21.189 +0900 [INFO] (0001:transaction): Logging initialized @2856ms
2022-07-01 20:17:21.867 +0900 [INFO] (0001:transaction): Acquired new access token
2022-07-01 20:17:21.868 +0900 [INFO] (0001:transaction): CALLING Account ID: 498-VPN-295 GET -> https://498-VPN-295.mktorest.com/rest/asset/v1/programs.json - params: {offset=[0], maxReturn=[200], filterType=[id], filterValues=[1635], batchSize=[300]}
2022-07-01 20:17:22.087 +0900 [INFO] (0001:transaction): CALLING Account ID: 498-VPN-295 GET -> https://498-VPN-295.mktorest.com/rest/v1/programs/members/describe.json - params: null
2022-07-01 20:17:22.343 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=16 / output tasks 8 = input tasks 1 * 8
2022-07-01 20:17:22.348 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
2022-07-01 20:17:22.402 +0900 [INFO] (pool-3-thread-1): Send bulk extract request [MarketoBulkExtractRequest{format='CSV', columnHeaderNames=null, filter={programId=1635}}]
2022-07-01 20:17:22.883 +0900 [INFO] (pool-3-thread-1): Acquired new access token
2022-07-01 20:17:22.883 +0900 [INFO] (pool-3-thread-1): CALLING Account ID: 498-VPN-295 POST -> https://498-VPN-295.mktorest.com/bulk/v1/program/members/export/create.json - params: null
2022-07-01 20:17:23.612 +0900 [INFO] (pool-3-thread-1): CALLING Account ID: 498-VPN-295 POST -> https://498-VPN-295.mktorest.com/bulk/v1/program/members/export/43976196-5140-4b47-bbcf-55a9bea02529/enqueue.json - params: null
2022-07-01 20:17:23.745 +0900 [INFO] (pool-3-thread-1): CALLING Account ID: 498-VPN-295 GET -> https://498-VPN-295.mktorest.com/bulk/v1/program/members/export/43976196-5140-4b47-bbcf-55a9bea02529/status.json - params: null
2022-07-01 20:17:23.868 +0900 [INFO] (pool-3-thread-1): Jobs [43976196-5140-4b47-bbcf-55a9bea02529] status is [Queued]
2022-07-01 20:18:23.868 +0900 [INFO] (pool-3-thread-1): CALLING Account ID: 498-VPN-295 GET -> https://498-VPN-295.mktorest.com/bulk/v1/program/members/export/43976196-5140-4b47-bbcf-55a9bea02529/status.json - params: null
2022-07-01 20:18:24.269 +0900 [INFO] (pool-3-thread-1): Jobs [43976196-5140-4b47-bbcf-55a9bea02529] status is [Completed]
2022-07-01 20:18:24.269 +0900 [INFO] (pool-3-thread-1): Total wait time ms is [60123]
2022-07-01 20:18:24.269 +0900 [INFO] (pool-3-thread-1): File size is [1540] bytes
2022-07-01 20:18:24.271 +0900 [INFO] (pool-3-thread-1): Download bulk export job [43976196-5140-4b47-bbcf-55a9bea02529]
2022-07-01 20:18:24.271 +0900 [INFO] (pool-3-thread-1): Range header value [bytes=0-]
2022-07-01 20:18:24.276 +0900 [INFO] (pool-3-thread-1): CALLING Account ID: 498-VPN-295 GET -> https://498-VPN-295.mktorest.com/bulk/v1/program/members/export/43976196-5140-4b47-bbcf-55a9bea02529/file.json - params: null
2022-07-01 20:18:24.418 +0900 [INFO] (pool-3-thread-1): Import data for program [1635], job_id [43976196-5140-4b47-bbcf-55a9bea02529] finish.[6] records imported/total [6]
false,,,,program_test_1_name,2022-07-01 09:51:38.000000 +0000,,,,false,2022-07-01 09:51:38.000000 +0000,2022-07-01 09:51:38.000000 +0000,,,,Influenced,,,,1635,true,1024889,2022-07-01 09:51:38.000000 +0000
false,,,,program_test_1_name,2022-07-01 09:51:39.000000 +0000,,,,false,2022-07-01 09:51:39.000000 +0000,2022-07-01 09:51:39.000000 +0000,,,,Influenced,,,,1635,true,1024890,2022-07-01 09:51:39.000000 +0000
false,,,,program_test_1_name,2022-07-01 09:51:39.000000 +0000,,,,false,2022-07-01 09:51:39.000000 +0000,2022-07-01 09:51:39.000000 +0000,,,,Influenced,,,,1635,true,1024936,2022-07-01 09:51:39.000000 +0000
false,,,,program_test_1_name,2022-07-01 09:51:40.000000 +0000,,,,false,2022-07-01 09:51:40.000000 +0000,2022-07-01 09:51:40.000000 +0000,,,,Influenced,,,,1635,true,1024937,2022-07-01 09:51:40.000000 +0000
false,,,,program_test_1_name,2022-07-01 09:51:40.000000 +0000,,,,false,2022-07-01 09:51:40.000000 +0000,2022-07-01 09:51:40.000000 +0000,,,,Influenced,,,,1635,true,1024938,2022-07-01 09:51:40.000000 +0000
false,,,,program_test_1_name,2022-07-01 09:51:40.000000 +0000,,,,false,2022-07-01 09:51:40.000000 +0000,2022-07-01 09:51:40.000000 +0000,,,,Influenced,,,,1635,true,1024939,2022-07-01 09:51:40.000000 +0000
2022-07-01 20:18:24.422 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2022-07-01 20:18:24.426 +0900 [INFO] (main): Committed.
2022-07-01 20:18:24.426 +0900 [INFO] (main): Next config diff: {"in":{},"out":{}}
```

**Review by kekekenta — APPROVED** [review]

File: build.gradle (hunk 3 above)
Comment (d-hrs) [reply build.gradle]: https://github.com/primenumber-dev/n-transfer-ui/blob/release/docs/how_to_embulk_plugin.md#fork%E5%BE%8C%E3%81%AE%E3%83%97%E3%83%A9%E3%82%B0%E3%82%A4%E3%83%B3%E3%81%AE%E3%83%90%E3%83%BC%E3%82%B8%E3%83%A7%E3%83%8B%E3%83%B3%E3%82%B0

にあるように`${forkしたバージョン}-trocco-${trocco内のバージョン}` でお願いします。
どうやら0.6.18からforkしているようです
https://github.com/trocco-io/embulk-input-marketo/blob/16e9d6653575a97e0d8de97b9e2205ddc193f24e/build.gradle#L19

なので、 `0.6.18-trocco-0.1.0` とかですかね…

またtagを切る際は、tagを直接ではなく https://github.com/trocco-io/embulk-input-marketo/releases/new からreleaseを作成してください。タグ名を入力して `Generate release notes` してもらえると。

Comment (t3t5u) [reply build.gradle]: 了解しました

なお fork 元の [v0.6.24](https://github.com/treasure-data/embulk-input-marketo/commits/v0.6.24) までをマージしたので [version は 0.6.24](https://github.com/trocco-io/embulk-input-marketo/blob/marge-treasure-data-v0.6.24/build.gradle#L18) にしてあります

ちなみに trocco 内のバージョンですが,
[後続の PR](https://github.com/trocco-io/embulk-input-marketo/pull/20/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R19) ではマイナーバージョンを増やしていくかんじでしょうか？
(trocco 内のバージョンは patchVersion だけにしようかと考えていました)

Comment (t3t5u) [reply build.gradle]: 対応しました
https://github.com/trocco-io/embulk-input-marketo/pull/19/commits/547063e0216bf98e3f8a19ea52bcf2b0c5c5199c

**Review by d-hrs — APPROVED** [review]

**Pull request merged**